charts/eks-anywhere-packages/templates/namespace.yaml

```diff
@@ -1,13 +1,21 @@
+{{- if not .Values.workloadOnly }}
 apiVersion: v1
 kind: Namespace
 metadata:
   annotations:
     "helm.sh/resource-policy": keep
   creationTimestamp: null
-  {{- if not .Values.workloadOnly }}
   name: eksa-packages
-  {{- else }}
+spec: {}
```

Comment:
I'd remove the spec and status if empty.

Reply:
Done. I didn't add those, but sure, can't hurt to remove them. Just extra noise in the review.